src/index.css

```
@@ -31,10 +31,28 @@
+  html, body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    min-height: 100vh;
+    background-color: #8A1538; /* Fallback color */
+    overflow-x: hidden;
+  }
   body {
     @apply bg-background text-foreground;
-    background: linear-gradient(135deg, #8A1538 0%, #6B0F2A 50%, #C9A54C 100%);
+    background: linear-gradient(135deg, #8A1538 0%, #6B0F2A 50%, #C9A54C 100%) fixed !important;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)